libethashseal/Ethash.cpp

```diff
@@ -84,10 +84,13 @@ StringHashMap Ethash::jsInfo(BlockHeader const& _bi) const
 
 EVMSchedule const& Ethash::evmSchedule(EnvInfo const& _envInfo) const
 {
-	if (_envInfo.number() >= chainParams().u256Param("frontierCompatibilityModeLimit"))
-		return HomesteadSchedule;
+	if (_envInfo.number() >= chainParams().u256Param("EIP150forkBlock"))
+		return DevconSchedule;
 	else
-		return FrontierSchedule;
+		if (_envInfo.number() >= chainParams().u256Param("frontierCompatibilityModeLimit"))
```

Comment:
Do we break anything if we change this to `homsteadForkBlock`? It would be a bit more consistent.